scripts/mint.ts

```diff
@@ -109,6 +109,50 @@ async function mintFor(tokenId: string, recipient: string) {
   );
 }
 
+async function mintForRange(tokenIdRange: string, recipient: string) {
+  const starsRecipient = toStars(recipient);
+
+  // Parse string from "1,10" -> "1" and "10"
+  let split = tokenIdRange.split(",");
+  let start = parseInt(split[0]), end = parseInt(split[1]);
+
+  // Verify proper range
+  if (isNaN(start) || start < 1) throw new Error("Invalid start ID");
+  if (isNaN(end) || end < 1) throw new Error("Invalid end ID");
+  if (start > end) throw new Error("Invalid range");
+
+  console.log('Minting tokens', start + '-' + end, 'for', starsRecipient);
+
+  // Loop through range and generate contract messages.
+  let msgArray = new Array();
+  for (let i = start; i <= end; i++) {
+    let msg = {
+      mint_for: { token_id: i, recipient: starsRecipient },
+    };
```

Comment:
Maybe you could add `JSON.stringify(msgArray, null, 2)`, then someone could copy and paste to sign with ledger

Reply:
`Minting tokens 37-39 for stars1f9y7sq4ujduk3k8swkfjfgdyy2t6p606zwtz3n
[
  {
    "typeUrl": "/cosmwasm.wasm.v1.MsgExecuteContract",
    "value": {
      "sender": "stars1f9y7sq4ujduk3k8swkfjfgdyy2t6p606zwtz3n",
      "contract": "stars1x8gwn06l85q0lyncy7zsde8zzdn588k2dck00a8j6lkprydcutwq0uyaaw",
      "msg": {
      "0": 123,
      ...
`

This is how it prints it. Is this OK?

Reply:
I'm not able to test this one yet since the site doesn't support it.

Reply:
Does "Execute a transaction" not work on testnet?

https://big-bang.publicawesome.dev/tx

Reply:
I don't think it works for batch transactions because the tx site is using client.execute(), which won't accept an array. Last time I tried putting a JSON array in the form I got an error.

Reply:
Confirmed this is a separate issue. Thanks for bringing this up.